.travis.yml

```diff
@@ -1,4 +1,3 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: PATH="./bats/bin:$PATH" test/run
```

Comment:
Yes, got that. I just ran `test/run` and it cloned the bats repo and ran the tests. Now need to figure out why it fails on Travis.